apps/shipments/serializers.py

```diff
@@ -265,25 +269,39 @@ class Meta:
 class TrackingDataSerializer(serializers.Serializer):
     payload = serializers.RegexField(r'^[a-zA-Z0-9\-_]+?\.[a-zA-Z0-9\-_]+?\.([a-zA-Z0-9\-_]+)?$')
 
-    def validate(self, attrs):
+    def validate(self, attrs):  # noqa: MC0001
```

Comment:
Generally let's not disable McCabe (code complexity analysis) without attempting to reduce complexity first. Not saying that the checker is perfect, but it is usually an indicator of room for improvement :+1: